eth-contracts/contracts/ServiceProviderFactory.sol

```diff
@@ -189,9 +189,9 @@ contract ServiceProviderFactory is InitializableV2 {
         );
 
         // Update min and max totals for this service provider
-        (uint typeMin, uint typeMax) = ServiceTypeManager(
+        (, uint typeMin, uint typeMax) = ServiceTypeManager(
             serviceTypeManagerAddress
-        ).getServiceTypeStakeInfo(_serviceType);
+        ).getServiceTypeInfo(_serviceType);
```

Comment:
should we reject if a user is attempting to register an invalid serviceType?

Reply:
this is already handled on line 152 above, but i'll add a test for this case as well